DirectXTex/DirectXTexDDS.cpp

```diff
@@ -2083,16 +2084,22 @@ HRESULT DirectX::SaveToDDSMemory(
                         return hr;
                     }
 
-                    size_t rowPitch = images[index].rowPitch;
+                    const size_t rowPitch = images[index].rowPitch;
```

Comment:
I should go through and 'constify`` a bunch of stuff, but I wanted to keep this diff under control so I just did a few around my other change.